test/acl_test.h

```diff
@@ -55,9 +55,9 @@ void CL_CALLBACK acl_test_notify_print(const char *errinfo,
 
 #define ACL_LOCKED(...)                                                        \
   do {                                                                         \
-    acl_lock();                                                                \
+    acl_mutex_wrapper.lock();                                                  \
     { __VA_ARGS__; }                                                           \
-    acl_unlock();                                                              \
+    acl_mutex_wrapper.unlock();                                                \
```

Comment:
The code in the test suite have setup and teardown functions which already somewhat setups RAII and there are also places which I can't create a local scope for RAII so I for uniformity I just used them like this throughout the code. Should this be changed? There are some places in the test suite that can use RAII and some places that cannot (unless further code changes are made).

Reply:
Yes. Also, there are some lock() and unlock() instances in the code where the critical section declares a local variable where it is used later out of the critical section. Again, this can be overcome by changing the declaration location but would require more effort.